src/components/ToastNotification.tsx

```diff
@@ -0,0 +1,57 @@
+import { Notification } from "@canonical/react-components";
+import { DefaultTitles } from "@canonical/react-components/dist/components/Notification/Notification";
+import { ToastNotificationType } from "context/toastNotificationProvider";
+import React, { FC } from "react";
+import { createPortal } from "react-dom";
+import Animate from "./Animate";
+
+interface Props {
+  notification: ToastNotificationType;
+  onDismiss: (notification?: ToastNotificationType[]) => void;
+  show: boolean;
+}
+
+const ToastNotification: FC<Props> = ({ notification, onDismiss, show }) => {
+  if (!notification) {
+    return;
+  }
+
+  return (
+    <>
+      {createPortal(
+        <Animate
+          show={show}
+          from={{
+            opacity: 0,
+          }}
+          to={{
+            opacity: 1,
+          }}
+          exitAnimation={[
+            { opacity: 1, transform: "translateY(0)" },
+            { opacity: 0, transform: "translateY(50px)" },
+          ]}
+          options={{ duration: 200 }}
```

Comment:
We need something like this class here as well.
```suggestion
      className="toast-notification-list-animate"

```

Reply:
Thanks David, I've done the fix for the single notification, will chat to @piperdeck about the LXD version prefix